test/cpp/tensorexpr/test_expr.cpp

```diff
@@ -0,0 +1,58 @@
+#include "test/cpp/tensorexpr/test_base.h"
+
+#include "test/cpp/tensorexpr/test_utils.h"
+#include "torch/csrc/jit/tensorexpr/buffer.h"
+#include "torch/csrc/jit/tensorexpr/ir.h"
+
+#include <cmath>
+#include <sstream>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+namespace torch {
+namespace jit {
+using namespace torch::jit::tensorexpr;
+
+void testExprVectorAdd01() {
```

Comment:
Most of the tests use IR evaluator, which is added in the subsequent commit, so the tests are also mostly added there.